.github/workflows/frontend_format.yaml

```diff
@@ -0,0 +1,18 @@
+name: frontend_format
+
+on:
+  push:
+    branches: [main, dev]
+
+jobs:
+  format:
```

Comment:
Can we add a `lint` job here if we aren't using a git hook like `husky` to do it locally on commit. We can also consider `lint-staged` for this

Reply:
Linting as a separate action would be better tbh. But yes, that makes sense.